include/eve/module/math/constant/egamma_sqr.hpp

```diff
@@ -0,0 +1,78 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core.hpp>
+
+namespace eve
+{
+  //================================================================================================
+  //! @addtogroup core
+  //! @{
+  //! @var egamma_sqr
+  //!
+  //! @brief Callable object computing the egamma_sqr constant value : square of the Euler-Mascheroni constant.
```

Comment:
Callable object computing the square of the [Euler-Mascheroni constant](@ref eve::egamma).